File: PRAKTIKUM PEMROGRAMAN I P1/MODUL 1/SOAL 4/PRAK104-2410817210002-Sebastian_Abe_Santoso.py
a = 400000; b = 350000; x = 13; y = 21; ax = a - a*x/100
by = b - b * y/100; ax = int(ax); by = int(by)
print("Harga sepatu A adalah", a, "\nHarga sepatu B adalah", b)
print(f"Sepatu A mendapat diskon {x:.0f}% sehingga harganya menjadi", ax)
print(f"Sepatu B mendapat diskon {y:.0f}% sehingga harganya menjadi", by)

File: PRAKTIKUM PEMROGRAMAN I P1/MODUL 1/SOAL 4/PRAK104-2410817210002-Sebastian_Abe_Santoso.c
#include <stdio.h>
int main(){
    int  a = 400000, b = 350000, x = 13, y = 21, ax = a - a*x/100, by = b - b * y/100;
    printf("Harga sepatu A adalah %d\nHarga sepatu B adalah %d\n", a,b);
    printf("Sepatu A mendapat diskon %d%% sehingga harganya menjadi %d\n", x, ax);
    printf("Sepatu B mendapat diskon %d%% sehingga harganya menjadi %d\n", y, by);
    return 0;
}